howl/client/howl_client.py

```diff
@@ -0,0 +1,102 @@
+from howl.context import InferenceContext
+import logging
+import time
+from typing import Callable
+
+import numpy as np
+import pyaudio
+import torch
+from howl.model.inference import InferenceEngine
+
+
```

Comment:
can you add some comments about what this class is for?

Reply:
maybe we should adopt the idea of docstring

Reply:
yeah still adding some code, will add docs after

Reply:
ok added some docs, let me know if anything is unclear to you